### PR TITLE
Fixed unknown 'exceptions' under Python3 (#49152)

### PR DESCRIPTION
**Pull request by erwindon**
### What does this PR do?
This PR fixes the (wheel)error.error command, which fails under python3.
The solution is a small code transplant from file `_compat.py`.

### What issues does this PR fix or reference?
This error was previously reported as #49152.

### Previous Behavior
Always exception about unknown identifier "exceptions"

### New Behavior
Behavior according to documentation.

### Tests written?
No
Tested using command `sudo salt HOSTNAME saltutil.wheel error.error "Exception" "This is an error."` after copying the updated file `salt/utils/error.py` into an installed system and restarting both the master and the minion of that system.
Manually verified that all `import exceptions` command are covered. I only found 2 cases: one in `_compat.py`, and this PR solves the issue for the other one (`salt/utils/error.py`).

### Commits signed with GPG?
No

### CODE REVIEW

File: salt/utils/error.py [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
 
-# Import python libs
-try:
+import sys
+# True if we are running on Python 3.
+PY3 = sys.version_info[0] == 3
+
+if PY3:
+    import builtins
+    exceptions = builtins
```
Comment (dwoz) [inline salt/utils/error.py]: This can be `from salt.ext.six.moves import builtins as exceptions`

**Review by dwoz — CHANGES_REQUESTED** [review]

File: salt/utils/error.py (hunk 1 above)
Comment (erwindon) [reply salt/utils/error.py]: @dwoz
Are you suggesting to just combine lines 13+14?
Then why not just `import builtins as exceptions`?
And should that also be applied to `_compat.py`?

Comment (dwoz) [reply salt/utils/error.py]: Using `salt.ext.six` also removes the need for the python 3 check. So 8-14 can become one line. Yes, the same can be done for lines 37-45 in `_compat.py`

Comment (erwindon) [reply salt/utils/error.py]: The code for `utils/error.py` is updated using this suggestion and re-tested in the same way as before.
I realize that changing `_compat.py` should not be done as part of this bugfix, as there is no bug there. So I did not update that one.

**Review by cachedout — APPROVED** [review]

**Comment by erwindon** [comment]
@cachedout I was just thinking on how to force the regression test to re-run as older/failing jobs had finished later and github got confused about that... your merge action just solved that 👍

**Review by dwoz — APPROVED** [review]

**Pull request merged**